CIScripts/TestIntegration.ps1

```diff
@@ -63,6 +63,7 @@ $SNATConfiguration = [SNATConfiguration] @{
     VMDir = $Env:SNAT_VM_DIR;
 }
 
+Test-VRouterAgentIntegration -Session1 $Sessions[0] -Session2 $Sessions[1] -TestConfiguration $TestConfiguration
```

Comment:
Why `Test-VRouterAgentIntegration` was moved to the beginning of the test suite?